src/chart/draw/consolidateData/transformData/makeNest.js

```diff
@@ -12,15 +12,19 @@ export default function makeNest(mark, entries, sublevel) {
         (this.config.x.type === 'linear' && this.config.x.bin) ||
         (this.config.y.type === 'linear' && this.config.y.bin)
     ) {
-        let xy = this.config.x.type === 'linear' && this.config.x.bin ? 'x' : 'y';
-        let quant = scale
+        const xy = this.config.x.type === 'linear' && this.config.x.bin ? 'x' : 'y';
+        mark.quant = scale
             .quantile()
-            .domain(extent(entries.map(m => +m[this.config[xy].column])))
+            .domain(
```

Comment:
I guess this is really the big change ... 

If I understand correctly, this is just to avoid overwriting user specified custom domains defined in a callback. If that's the case, it makes perfect sense to me! (If there is some other use case here, lmk).

Reply:
Yep, the bin width is calculated with the x-domain rather than with the extent of the visible results.  Useful when comparing the full distribution with the distribution of subsets.